### PR TITLE
CSS State Selector

### PR DESCRIPTION
**Pull request by daviddarnes**
- [x] Switch to using CSS state selector from attribute selector, except with multiple elements with IDs
- [x] Update demo to match with new changes
- [x] Update docs
- [x] Bump version

### CODE REVIEW

**Review by keithamus — APPROVED** [review]
LGTM

File: demo.html [hunk 1]
```diff
@@ -1,42 +1,48 @@
...
+  </head>
+  <body>
+    <h2>Single element demo</h2>
+    <is-playing>
+      <audio controls src="https://darn.es/sounds/daviddarnes.m4a"></audio>
+    </is-playing>
+
+    <hr />
+
+    <h2>Multiple elemnents example, with unique IDs</h2>
```
Comment (mxdvl) [inline demo.html]:
```suggestion
    <h2>Multiple elements example, with unique IDs</h2>
```

Comment (daviddarnes) [reply demo.html]: Oh thanks!!

**Comment by daviddarnes** [comment]
Wasn't expecting comments, but very much appreciate them @keithamus and @mxdvl!

**Pull request merged**